packages/kit/src/runtime/load.js

```diff
@@ -4,10 +4,19 @@
  */
 export function normalize(loaded) {
 	// TODO should this behaviour be dev-only?
+	const hasStatusError =
+		loaded.status && loaded.status >= 400 && loaded.status <= 599 && !loaded.redirect;
+	if (loaded.error || hasStatusError) {
+		const status = loaded.status;
+
+		if (!loaded.error && hasStatusError) {
+			return {
+				status,
+				error: new Error('Internal Server Error')
```

Comment:
I think I'd lean towards just doing `new Error()` and leaving the message empty

Reply:
Changed it to be an empty message